### ISSUE TITLE
Organise project catch up with Lesedi

### ISSUE THREAD

**Issue by raquel-ucl** [namibmap/IPPR#2]
Lesedi wants to talk about project progress, new server details and moving forward.

- [x] Send doodle poll
- [x] Decide on a date
- [x] Organise agenda

**Comment by raquel-ucl** [comment]
[Doodle poll](http://doodle.com/poll/inxa6uwnytg72gfg) sent.

Draft agenda:

```
1) Progress report 
- Karen's mockups
- Marta's DB work
- Raquel's CartoDB work
2) Plan next steps and priorities
3) Server details from Lesedi 
4) CartoDB Q&A for Lesedi sharing screens (maybe separate call?)

Please add more stuff if needed.
```

**Comment by raquel-ucl** [comment]
Call done. Notesto be published in #4
